gui/src/installer/view.rs

```diff
@@ -951,9 +983,13 @@ pub fn define_bitcoin<'a>(
                     .spacing(10)
                     .push(Container::new(
                         button::secondary(None, "Check connection")
-                            .on_press(Message::DefineBitcoind(
-                                message::DefineBitcoind::PingBitcoind,
-                            ))
+                            .on_press_maybe(if check_connect_enable {
```

Comment:
I like this `on_press_maybe`! We'll be able to use that in many places :)

Reply:
i just noticed i have never been use in Liana! :smile:

Reply:
Apparently it was only added in iced 0.10.0 :monocle_face: